test/conformance/queue/urQueueGetInfo.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+#include <uur/fixtures.h>
+
+using urQueueGetInfoTest = uur::urQueueTestWithParam<ur_queue_info_t>;
+
+UUR_TEST_SUITE_P(urQueueGetInfoTest,
+                 ::testing::Values(
+
+                     UR_QUEUE_INFO_CONTEXT,         //
+                     UR_QUEUE_INFO_DEVICE,          //
+                     UR_QUEUE_INFO_DEVICE_DEFAULT,  //
+                     UR_QUEUE_INFO_PROPERTIES,      //
+                     UR_QUEUE_INFO_REFERENCE_COUNT, //
+                     UR_QUEUE_INFO_SIZE             //
```

Comment:
How does `clang-format` deal with this change?

```suggestion
                     UR_QUEUE_INFO_CONTEXT,
                     UR_QUEUE_INFO_DEVICE,
                     UR_QUEUE_INFO_DEVICE_DEFAULT,
                     UR_QUEUE_INFO_PROPERTIES,
                     UR_QUEUE_INFO_REFERENCE_COUNT,
                     UR_QUEUE_INFO_SIZE,
```

I might look less weird.

Reply:
It does this
```cpp
UUR_TEST_SUITE_P(urQueueGetInfoTest,
                 ::testing::Values(UR_QUEUE_INFO_CONTEXT, UR_QUEUE_INFO_DEVICE,
                                   UR_QUEUE_INFO_DEVICE_DEFAULT,
                                   UR_QUEUE_INFO_PROPERTIES,
                                   UR_QUEUE_INFO_REFERENCE_COUNT,
                                   UR_QUEUE_INFO_SIZE),
                 uur::deviceTestWithParamPrinter<ur_queue_info_t>);
```
The reason I added the comments was to have one enum per line, but I think this looks fine.

Reply:
done

Reply:
I think this misses the point I was making which is to have a trailing `,` after the last item in the list. It changes the behaviour of `clang-format`.

Reply:
Although I just tried it and it did not do what I expected. Perhaps some interaction with the macro. Nevermind.

Reply:
Yeah it doesn't like trailing commas - I had to remove it to get it to compile